tasks/bloom-etl/src/etl/extract/extractor.ts

```diff
@@ -0,0 +1,72 @@
+import { Axios } from "axios"
+import { Jurisdiction, Listing, Response, UrlInfo } from "../../types"
+import { ExtractorInterface } from "./extractor-interface"
+import { BaseStage } from "../base-stage"
+
+export class Extractor extends BaseStage implements ExtractorInterface {
+  axios: Axios
+  urlInfo: UrlInfo
+  jurisdictions: Array<Jurisdiction>
+
+  constructor(axios: Axios, urlInfo: UrlInfo, jurisdictions: Array<Jurisdiction>) {
+    super()
+    this.axios = axios
```

Comment:
I used dependency injection here because the jest mocks weren't acting reliably and I needed a way to force the Extractor to use a specific instance of "axios", even if it wasn't axios proper.

Reply:
Added